mlflow/gateway/providers/anthropic.py

```diff
@@ -38,9 +38,14 @@ async def completions(self, payload: completions.RequestPayload) -> completions.
             )
 
         payload = rename_payload_keys(
-            payload, {"max_tokens_to_sample": "max_tokens", "stop_sequences": "stop"}
+            payload, {"max_tokens": "max_tokens_to_sample", "stop": "stop_sequences"}
         )
 
+        if not payload.get("stop_sequences", None):
+            # NB: A value of None will throw in Anthropic's endpoint. Not applying this
+            # defaulted parameter will revert to the default value of "\n\nHuman:"
+            payload.pop("stop_sequences")
```

Comment:
Instead, we can use `jsonable_encoder(payload, exclude_none=True)`.

Reply:
ah, great point! Changing :)